docs/src/pages/guides/cli-options.md

```diff
@@ -320,6 +320,14 @@ monika --summary
 Please note that you need to run the above command from the same working directory as the running monika you want to see/check.
 The '--summary' flag also will not work when no other monika process is running.
 
+## TLS Reject Unauthorized
+
+If there is a probe with request(s) that uses HTTPS, Monika will show an error if the target's TLS certificate is invalid. You can configure whether HTTPS requests should ignore invalid certificates using the `--ignoreInvalidTLS` flag.
+
```

Comment:
do we know what kind of "invalid" certificates are occurring? can we be more specific if possible? All invalid certs will be ignored?

Reply:
Accordin to the [NodeJS docs](https://nodejs.org/api/tls.html#tlsconnectoptions-callback):

>rejectUnauthorized [<boolean>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#Boolean_type) If not false, the server certificate is verified against the list of supplied CAs. An 'error' event is emitted if verification fails; err.code contains the OpenSSL error code. Default: true.

Some of the problems occurred is, but not limited to:
- Error: certificate has expired
- DEPTH_ZERO_SELF_SIGNED_CERT Error: self-signed certificate
- ERR_TLS_CERT_ALTNAME_INVALID Error [ERR_TLS_CERT_ALTNAME_INVALID]: Hostname/IP does not match certificate's altnames: Host: somehost.com. is not in the cert's altnames: DNS:*.dns.any, DNS:dns.any
- Error: write EPROTO 589B604DD17F0000:error:0A00010B:SSL routines:ssl3_get_record:wrong version number:../deps/openssl/openssl/ssl/record/ssl3_record.c:354:

And many more that I didn't know much about it.

Reply:
I do found some research though from ChatGPT but it seems that some of them differ from the Node.js ones

https://github.com/openssl/openssl/blob/master/crypto/err/openssl.txt

Reply:
Found the docs. Updated the commit